src/packets.rs

```diff
@@ -766,6 +767,15 @@ pub struct WinChange {
     pub height: u32,
 }
 
+/// An environment variable
+#[derive(Debug, SSHEncode, SSHDecode)]
+#[cfg_attr(feature = "arbitrary", derive(Arbitrary))]
+pub struct Environment<'a> {
+    // pub wants_reply: bool, // TODO: needed?
```

Comment:
`wants_reply` isn't needed here, it's part of the `ChannelRequest` struct.

Reply:
Oh, sorry, I saw it listed in https://datatracker.ietf.org/doc/html/rfc4254#section-6.4 and blindly added it there (trying to follow RFC/spec to the letter).